src/test/resources/suite.epam.tc.hw2/HomePageOneTestSuite

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+<suite name="Home Page One Test Suite">
+  <test name="Home Page One">
+    <classes>
+      <class name="com.epam.tc.hw2.ex1.HomePageOneTest"/>
+    </classes>
+  </test>
+</suite>
```

Comment:
Add a new blank line. Did you run your project via "mvn test"?

Reply:
The blank line is added. Yes I did. The test is running successfully.

Reply:
I told about a new line at the end of the file. Don't you run your code through the "mvn test" command?